src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({ name }) {
+  return (
+    <>
+      <button type="button">{ name }</button>
+    </>
+  );
+}
```

Comment:
[Optional]

Consider using ES6 arrow function instead. With arrow function, you don't need an implicit `return` keyword for this component.


```jsx
export default Button = ({ name }) => (
      <button type="button">{ name }</button>
 );
```

Also, is there a reason why you are using an empty tag in this component? :thinking: